### ISSUE TITLE
Build NRRD Site Map

### ISSUE THREAD

**Issue by daniel-coleman-1** [ONRR/nrrd#1810]

**Comment by daniel-coleman-1** [comment]
5.0.3

**Comment by daniel-coleman-1** [comment]
@Maroyafaied this is ready for your review

**Comment by mcharg** [comment]
@daniel-coleman-1 I reviewed Explore Data and Query Data and added some comments about how to organize them.  Let me know if you want to jump on a call to discuss.

**Comment by daniel-coleman-1** [comment]
@mcharg I've implemented changes in Explore Data, but left out numbers on the lower portion.  Hoping you have some guidance on setting numbers with this structure.

https://vp0fb7.axshare.com

**Comment by mcharg** [comment]
@daniel-coleman-1 I reviewed Explore Data > Revenue.  You should be able to extrapolate those comments to Production and Disbursements.

**Comment by daniel-coleman-1** [comment]
@mcharg and @Maroyafaied thank you for the notes, I've updated it in a manner that I think best represents what you're thinking.  Could you take another look?

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 I reviewed the homepage and listed my comments, here is a summary of my comments.

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 sorry, I didn't finish my comment but I reviewed the homepage, it's looking good. Here are a summary of my comments
1. The "source" breakout is missing under revenue>yearly>fiscal year. It should be 0.2.1.1.1 before revenue type.
2.  0.3.1.2 should be called Recipient instead of revenue
3. I tagged @mcharg in the comments but I'm thinking the source files don't need to be listed multiple times since they don't change once you select yearly or monthly for each data type.  Also the numbering under each source file need to change , here is what they should be.  
- Revenue Yearly and Revenue Monthly (3.3)
- Disbursements  Yearly (3.6), Disbursements Monthly (3.5)
- Production Yearly and Monthly (3.1)

**Comment by mcharg** [comment]
@daniel-coleman-1 I took another pass through Explore Data.  There are a number of my previous comments that weren't resolved.

**Comment by daniel-coleman-1** [comment]
@mcharg can we jump on a call to discuss?  I know you're heading out for the weekend soon so Monday morning works

**Comment by mcharg** [comment]
@daniel-coleman-1 Sorry, I'd already logged off for the day on Friday.  I sent you an invite for a call this morning.

**Comment by daniel-coleman-1** [comment]
@mcharg thanks for chatting earlier, I have updated the sitemap with the changes we discussed:
https://vp0fb7.axshare.com

**Comment by mcharg** [comment]
@daniel-coleman-1 I took another pass through Explore Data and Query Data.  You need to go through the site with a fine toothed comb and see what changes when you interact.  That's all I'm doing when I review and I shouldn't have to be documenting as many things as I am.

**Comment by daniel-coleman-1** [comment]
@Maroyafaied this is ready for review.

**Comment by Maroyafaied** [comment]
Thanks I'll review, @mcharg can you review the query and explore data.

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 I only looked at the homepage, these comments highlighted still need to be addressed that I mentioned in my previous review. 
![image](https://user-images.githubusercontent.com/40175688/146095847-c7319726-a005-4184-aefb-d2dbc26ca3f2.png)

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 I'm responding to your comment on the nrrd-git-feed channel, again please comment on the actual issue and not in that channel.  Here are pics of what I'm referring to, we can meet tomorrow if this still doesn't make sense. 

Under fiscal year revenue, you need to add a box for the "Source" breakout like you have under calendar year.  It needs to go before revenue type like you have it for calendar year.
![image](https://user-images.githubusercontent.com/40175688/146097183-23376c1a-c3b1-4387-a823-c9b215698b97.png)

All the source files for monthly disbursements should be 3.5 not 3.6
![image](https://user-images.githubusercontent.com/40175688/146097088-19c5a13a-7cdf-4502-a4ea-d8ff96dff77e.png)

**Comment by daniel-coleman-1** [comment]
@Maroyafaied ahhhh ok I understand what you're saying

**Comment by Maroyafaied** [comment]
@daniel-coleman-1 I made a comment on 3.0 Download Data to add a box for this historical revenue data we just added to the website and the files. Please let me know when that update is made and when the homepage is ready for review again. ![image.png](https://images.zenhubusercontent.com/5c58acad07cc4d6f39e54f67/521e9e1d-23ea-4664-b9ea-ff2c5877c654)

**Comment by mcharg** [comment]
@daniel-coleman-1 It doesn't look like you resolved any of my comments on Explore Data and I noted which are still outstanding for Query Data.

**Comment by daniel-coleman-1** [comment]
@mcharg I may not be understanding what you're trying to communicate, I'll take another look later today and may schedule some time for us to talk tomorrow to get on the same page

**Comment by mcharg** [comment]
@daniel-coleman-1 We've already walked through it and you didn't make the changes we talked about, so I added them as comments and you still didn't make the changes.  Not sure how much clearer I can be.

**Comment by daniel-coleman-1** [comment]
@mcharg yes seems like a communication breakdown.  Let's get some time to talk.

**Comment by daniel-coleman-1** [comment]
ok @mcharg this is ready for a (hopefully final) review

**Comment by mcharg** [comment]
@daniel-coleman-1 It'll likely be Monday before I get a chance to review.  I've got a lot going on today and you told me you'd have this ready to review by EOD Wednesday, so I didn't plan on having to review it today.

**Comment by mcharg** [comment]
@daniel-coleman-1 I reviewed the revenue sections for both Explore Data and Query Data.  Most of what we discussed in our last meeting wasn't correctly addressed.

**Comment by Maroyafaied** [comment]
@mcharg @daniel-coleman-1 I'm going to set up a meeting to discuss and to decide how to move forward with this task.
